src/rust/engine/ui/src/display.rs

```diff
@@ -338,7 +380,9 @@ impl EngineDisplay {
   // Initiates one last screen render, then terminates the EngineDisplay and returns the cursor
   // to a static position, then prints all buffered stdout/stderr output.
   pub fn finish(&mut self) {
-    self.render();
+    //We don't care about handling the output from render() here, since we're about to shut down
```

Comment:
In future, include spaces after `//` comments would align a bit better with the rest of the codebase.